pkg/operation/botanist/waiter.go

```diff
@@ -21,12 +21,12 @@ import (
 	"sync"
 	"time"
 
+	resourcesv1alpha1 "github.com/gardener/gardener-resource-manager/pkg/apis/resources/v1alpha1"
```

Comment:
Why is this changed?

Reply:
Not my change. May be simple reordering due to default formatting. Will check.